src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager/HighestNotAbstainedVoterAccessDecisionManager.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Symfony\Component\Security\Core\Authorization\AccessDecisionManager;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+/**
+ * Grants access depending on the first not abstained voter decission.
```

Comment:
Yes, I like the idea of calling it `FirstNotAbstainedVoterAccessDecisionManager` maybe it makes it clearer on what it does.